### DISCUSSION TITLE
cant create fold with current foldmethond.

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#2135] [Q&A] [answered]
When I run fold command like zfat or zfap in , I get the error message .

I don't know what other info should I provide. How to fix this issue?

**Comment by dpetka2001** [comment]
What Neovim version are you using?

**Reply by OriLiMu** [reply]
Version: 
NVIM v0.9.4
Build type: Release
LuaJIT 2.1.1699392533

![image](https://github.com/LazyVim/LazyVim/assets/11295355/9b7f3752-0b41-4251-a752-b64fe7c49683)

**Reply by dpetka2001** [reply, marked as answer]
Put `vim.opt.foldmethod = "manual"` in your `/lua/config/options.lua` file.
